### ISSUE TITLE
Bad formatting of a long list of arguments in :is(), :not()

### ISSUE THREAD

**Issue by wenfangdu** [prettier/prettier#10486]
**Prettier 2.2.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEALGBbANgbQCYCGMBAtJFhAE4kYR5wC8A5IZQNZMC6ABEgJYBnABQAdKN15QIMUeIncCAGm5j5C7gCplq+QDoARjCgksfKG21yJBoybNtNltbohgYfSMbzSSAMz5YWE56ru6eJN4wfgFY3AAORKjBEjhgWAQCAgB6zHFYJFw6AJTBunlkYkXcwDrkVEjcAG4ElEIkZBAU1PAAHlFxlHwYLQCeRQDcYgC+IIogEHHu0ALIoC2UEADuAAotCCsoBFibBCMrc-qUBGBscDAAyglgZgDmyDCUAK5wc3AY+nA8PQ8AAZAhQF6fAgvOAAMSowxg7ghyBABE+MAgszQmCwAHVUHx4AInnB7vsiXxGkSRqiwJlsWYBHBKDBtlcXsNkL4jsy5gArAQ9ABCVxud3uBAwcBBZjg3N5PxAgp691eWDgAEVPtJ5UgeVg+SAEpRmZQ6Qy5gMzDA8Xw8DBUMgABwABitG2ZeKucVRAzgZsa8rmAEcdfB2QsDmiBCQoHBAYDsZQ4GG+Cn2dCufrFXNmRg+O8vkqBOqteG9R9vnNiPo7Q6nUgAEw1q4BV4AYQgGGzIADAFZsZ9mQAVAj6A4Go2Nb4ASSg9Fg9zAg0WAEEF-cYCMNQrDXAplMgA)

```sh
--parser css
--single-quote
```

**Input:**

```css
html[data-color-mode='dark'] :is(
  :not(
    a, 
    a *, 
    .btn-link, 
    .btn-link *, 
    .octicon-dot-fill, 
    .octicon-dot-fill path, 
    [class^='pl-']
  ), 
  .pl-c
) {
  color: var(--color-text-primary);
}
```

**Output:**

```css
html[data-color-mode='dark']
  :is(:not(a, a *, .btn-link, .btn-link *, .octicon-dot-fill, .octicon-dot-fill
      path, [class^='pl-']), .pl-c) {
  color: var(--color-text-primary);
}

```
**Expected behavior:**
```css
html[data-color-mode='dark'] :is(
  :not(
    a, 
    a *, 
    .btn-link, 
    .btn-link *, 
    .octicon-dot-fill, 
    .octicon-dot-fill path, 
    [class^='pl-']
  ), 
  .pl-c
) {
  color: var(--color-text-primary);
}
```

**Comment by khalilazaiez** [comment]
Is there any extension that need to be installed to perform this task or is it just code?

**Comment by wenfangdu** [comment]
@khalilazaiez I'm not sure what you mean, for the reproduction case, please use the "playground link" in the description.

**Comment by wachunga** [comment]
Browser support for :is and :where are quite good now, so I expect these will become much more common
https://developer.mozilla.org/en-US/docs/Web/CSS/:is

**Comment by fisker** [comment]
Fixed by #13577
